account_mtd/models/res_users.py

```diff
@@ -73,8 +73,8 @@ def _get_vendor_headers(self, vendor_ip):
         return {
             "Gov-vendor-version":'Odoo%20SA=13.0&servercode=Python3.0',
             'Gov-vendor-license-ids': urllib.parse.urlencode({'Odoo':'13.0-20201006-community-edition'}),
-            "Gov-vendor-public-ip": vendor_ip,
-            "Gov-vendor-forwarded":'by='+vendor_ip+'&for='+self.client_ip,
+            "Gov-vendor-public-ip": str(vendor_ip),
```

Comment:
I think is better to use .format in these cases
```suggestion
            "Gov-vendor-public-ip": '{}'.format(vendor_ip),
```